src/xrpld/rpc/handlers/LedgerEntry.cpp

```diff
@@ -806,8 +806,50 @@ parseMPToken(Json::Value const& mptJson, Json::Value& jvResult)
     }
 }
 
+static std::optional<uint256>
+parsePermissionedDomains(Json::Value const& pd, Json::Value& jvResult)
+{
+    if (pd.isString())
+    {
+        Json::Value result;
+        auto const index = parseIndex(pd, result);
+        if (!index)
+            jvResult[jss::error] = "malformedIndex";
```

Comment:
This error is confusing - there's no field called `index` in this case.

Reply:
renamed to ObjectId